include/eve/detail/function/reverse.hpp

```diff
@@ -0,0 +1,39 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+#include <array>
+
+namespace eve
+{
+  EVE_MAKE_CALLABLE(reverse_, reverse);
+
```

Comment:
We usually provide a pattern generator too.

Reply:
Oh - I saw that. What's the point?

Reply:
so we have a shortcut for user `v[ reverse_n<8> ]`

Reply:
Discussed. I think we shoul delete it everywhere.